cake-recipe.txt

```diff
@@ -7,7 +7,7 @@ Ingredients:
  - 1 teaspoon salt
  - 2 eggs
  - 1 cup milk
- - 1/2 cup vegetable oil
+ - 1/2 cup Dr.Raj's oil
```

Comment:
Isn't Dr.Raj dead? Even if he is still alive, Is he still into the oil business at old age? :-)